internal/api/mail.go

```diff
@@ -558,6 +562,26 @@ func validateEmail(email string) (string, error) {
 	if err := checkmail.ValidateFormat(email); err != nil {
 		return "", badRequestError(ErrorCodeValidationFailed, "Unable to validate email address: "+err.Error())
 	}
+
+	email = strings.ToLower(email)
+
+	if len(a.config.External.Email.AuthorizedAddresses) > 0 {
+		// allow labelled emails when authorization rules are in place
+		normalized := emailLabelPattern.ReplaceAllString(email, "@")
+
+		found := false
+		for _, authorizedAddress := range a.config.External.Email.AuthorizedAddresses {
+			if normalized == authorizedAddress {
+				found = true
```

Comment:
/nit Can consider early return
```suggestion
				return email, nil 
```

The current approach is fine as well though.